src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Controls/RowDefinitionCollectionConverter.cs

```diff
@@ -0,0 +1,117 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel;
+using System.Windows.Controls;
+using System.Windows.Markup;
+using System.Globalization;
+using MS.Internal;
+
+#pragma warning disable 1634, 1691  // suppressing PreSharp warnings
```

Comment:
Unless you guys are still using PreSharp (don't find that likely), I don't see why we need to suppress a warning about suppressing warnings.